modules/data/src/main/java/io/fluo/webindex/data/spark/IndexEnv.java

```diff
@@ -110,14 +118,23 @@ public void initAccumuloIndexTable(SortedSet<Text> splits) {
     }
 
     try {
-      conn.tableOperations().addSplits(dataConfig.accumuloIndexTable, splits);
+      conn.tableOperations().addSplits(table, splits);
     } catch (AccumuloException | AccumuloSecurityException | TableNotFoundException e) {
       throw new IllegalStateException("Failed to add splits to Accumulo table " + table, e);
     }
   }
 
-  public void initAccumuloIndexTable() {
-    initAccumuloIndexTable(getDefaultSplits());
+  public void setFluoTableSplits(SortedSet<Text> splits) {
+    final String table = fluoConfig.getAccumuloTable();
+    if (conn.tableOperations().exists(table)) {
```

Comment:
check is not really needed.  Split will throw TableNotFound if it does not exists.